modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/SampleSQLBuilder.java

```diff
@@ -318,28 +351,80 @@ protected void mergeSQL(File inputFile, FileChannel outputFileChannel)
 		inputFile.delete();
 	}
 
-	protected void writeToInsertSQLFile(
-			File dir, String tableName, Map<String, Writer> insertSQLWriters,
-			String insertSQL)
+	protected void writeToSQLFile(
+			File dir, String tableName, Map<String, Writer> sqlWriters,
+			String sql)
 		throws IOException {
 
-		Writer insertSQLWriter = insertSQLWriters.get(tableName);
+		Writer writer = sqlWriters.get(tableName);
 
-		if (insertSQLWriter == null) {
+		if (writer == null) {
 			File file = new File(dir, tableName + ".sql");
 
-			insertSQLWriter = createFileWriter(file);
+			writer = createFileWriter(file);
 
-			insertSQLWriters.put(tableName, insertSQLWriter);
+			sqlWriters.put(tableName, writer);
 		}
 
-		insertSQLWriter.write(insertSQL);
+		writer.write(sql);
 	}
 
+	private void _loadCreateSQL(InputStream inputStream, Writer writer)
+		throws IOException {
+
+		try (BufferedReader reader = new BufferedReader(
+				new InputStreamReader(inputStream))) {
+
+			String line;
+
+			while ((line = reader.readLine()) != null) {
+				writer.append(line);
+				writer.append(System.lineSeparator());
+			}
+		}
+	}
+
+	private void _loadCreateSQLsFromClassLoader(Writer writer)
```

Comment:
@slnn Please rename this to **_loadCreateSQLs**, the method name is not consistent with the parameter, when you say load from class loader, usually you would have a class loader parameter, and here you actually do not need a class loader parameter as you always use the same one.

Tina